streaming/vision/convert/coco.py

```diff
@@ -181,15 +181,15 @@ def each(dataset: _COCODetection, shuffle: bool) -> Iterable[Dict[str, bytes]]:
         img_data = dataset.images[img_id]
         img_basename = img_data[0]
         img_filename = os.path.join(dataset.img_folder, img_basename)
-        img_bytes = open(img_filename, 'rb').read()
+        img = Image.open(img_filename)
 
         yield {
-            'img': img_bytes,
-            'img_id': np.int64(img_id).tobytes(),
-            'htot': np.int64(htot).tobytes(),
-            'wtot': np.int64(wtot).tobytes(),
-            'bbox_sizes': bbox_sizes.numpy().tobytes(),  # (_, 4) float32.
-            'bbox_labels': bbox_labels.numpy().tobytes(),  # int64.
+            'img': img,
+            'img_id': img_id,
+            'htot': htot,
+            'wtot': wtot,
+            'bbox_sizes': bbox_sizes,  # (_, 4) float32.
+            'bbox_labels': bbox_labels,  # int64.
```

Comment:
maybe fix comment to "# list of int64" or such

Reply:
It is correct. For example, below is the execution output from the same script

```
(Pdb) bbox_size
(0.0, 0.105375, 0.911296875, 0.9311875)
(Pdb) bbox_label
58
```